src/main/java/eventManagement/RestAPI.java

```diff
@@ -52,6 +53,37 @@ public void deleteEvent(@PathVariable("id") int id) {
 		eventService.delete(id);
 	}
 	
+	@CrossOrigin
+	@GetMapping("/customers")
+	public Collection<Customer> getAllCustomers() {
+		return customerService.getCustomers();
+	}
+	
+	@CrossOrigin
+	@GetMapping("/customer/{id}")
+	public Customer getCustomer(@PathVariable("id") int id) {
+		return customerService.getCustomerById(id);
+	}
+	
+	@CrossOrigin
+	@PostMapping("/customers")
+	public Customer addCustomer(@RequestBody Customer customer) {
+		customerService.createCustomer(customer);
+		return customer;
+	}
+	
+	@CrossOrigin
+	@PutMapping("/customer/{id}")
+	public Customer updateCustomer(@PathVariable("id") int id, Customer customer) {
+		customerService.update(id, customer);

```

Comment:
Here we call "update" method with the two parameters. CustomerService.java should implement the method.

Reply:
@lyn-Jia

Reply:
gotcha

Reply:
Not sure what you mean here.